modules/javafx.graphics/src/main/java/javafx/scene/layout/AnchorPane.java

```diff
@@ -182,9 +182,9 @@ public static void setBottomAnchor(Node child, Double value) {
     }
 
     /**
-     * Returns the child's bottom anchor constraint if set.
+     * Returns the child's bottom anchor constraint, if set.
      * @param child the child node of an anchor pane
-     * @return the offset from the bottom of the anchor pane or null if no bottom anchor was set
+     * @return the offset from the bottom of the anchor pane, or {@code null} if no bottom anchor was set
```

Comment:
minor suggestion: move explanation of when it returns null to the description,  i.e.:
```

Returns the child's bottom anchor constraint, if set, otherwise returns {@code null}.

@return the offset from the bottom of the anchor pane, or {@code null}
```

Reply:
I don't mind either way.

Reply:
`@return` statements usually state this condition explicitly, like `Map#get`: "the value to which the specified key is mapped, or `null` if this map contains no mapping for the key".

Reply:
My point is that it should be mentioned in the first paragraph then.  It can be repeated in @return later, if you prefer.  Map.get() does indeed mention this in the first paragraph:

![Screenshot 2024-02-23 at 14 30 02](https://github.com/openjdk/jfx/assets/107069028/977a6dbc-c0b0-4a00-a14d-8ff40353c365)

Reply:
Given that the changes are so far limited to spelling and formatting (code style for null), I think it would be fine to limit the changes in this PR to what he has already done.